cli/src/pcluster/validators/directory_service_validators.py

```diff
@@ -37,7 +37,8 @@ def _validate(self, domain_addr, additional_sssd_configs):
         elif domain_addr_scheme == "ldap":
             warning_message = "The use of the ldaps protocol is strongly encouraged for security reasons."
             tls_disabled = (
-                additional_sssd_configs.get("ldap_auth_disable_tls_never_use_in_production", "false").lower() == "true"
+                str(additional_sssd_configs.get("ldap_auth_disable_tls_never_use_in_production", "false")).lower()
```

Comment:
We should include a unit test to cover the bug.

Reply:
Added in next rev